src/main/java/io/cryostat/util/IllegalArgumentExceptionMapper.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The Cryostat Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.cryostat.util;
+
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.ext.ExceptionMapper;
+import jakarta.ws.rs.ext.Provider;
+
+@Provider
+public class IllegalArgumentExceptionMapper implements ExceptionMapper<IllegalArgumentException> {
+    @Override
+    public Response toResponse(IllegalArgumentException exception) {
+        return Response.status(400).build();
+    }
```

Comment:
There is also this class that this behaviour could be merged into:

https://github.com/cryostatio/cryostat3/blob/main/src/main/java/io/cryostat/ExceptionMappers.java

I did it this way just so that it's all consolidated into one file for ease of reference.

Reply:
I missed it, will fix.